File: EpiFib/mobile/EpiFib/EpiFib/EmergencyContainer.swift
//
//  EmergencyContainer.swift
//  EpiFib
//
//  Created by Gregory Hewett on 3/19/17.
//  Copyright © 2017 The Reach Lab, LLC. All rights reserved.
//

import UIKit
import EVReflection

final class EmergencyContainer: EVNetworkingObject
{
    var deviceId: String?
    
    var emergencyTypesSupported: [EmergencyType]?
    
    var location: ExactLocation?
    
    convenience init(_ emergencyTypesSupported: [EmergencyType], exactLocation: ExactLocation)
    {
        self.init()
        self.emergencyTypesSupported = emergencyTypesSupported
        self.location = exactLocation
    }
}

File: EpiFib/www/src/Common/Models/EmergencyContainer.cs
// <copyright file="EmergencyContainer.cs" company="The Reach Lab, LLC">
//     The Reach Lab, LLC. All rights reserved.
// </copyright>
// <author>Gregory Hewett</author>

namespace Common.Models
{
    using System.Collections.Generic;

    public class EmergencyContainer
    {
        public string DeviceId { get; set; }

        public IEnumerable<EmergencyType> EmergencyTypesSupported { get; set; }

        public ExactLocation Location { get; set; }
    }
}
